applications/ultrasound_postprocessing/ultra_post/app/holoscan_app.py

```
@@ -0,0 +1,306 @@
+
+    app = USPPApp()
+    # Enable Data Flow Tracking and save logs
+    print(f"Initializing Tracker with log file: {args.log}")
+    try:
+        with Tracker(app, filename=args.log) as tracker:  # noqa: F841
+            print("Tracker started. Running app...")
+            app.run()
+            print("App run finished.")
+    except KeyboardInterrupt:
+        print("User interrupted.")
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)